run-qa-checks

```diff
@@ -15,14 +15,10 @@ rm -rf ./openwisp_controller/geo/tests/.pytest_cache \
 jshint openwisp_controller/pki/static/admin/pki/js/*.js
 jshint openwisp_controller/config/static/config/js/*.js
 jshint openwisp_controller/connection/static/connection/js/*.js
-stylelint "openwisp_controller/config/static/config/css/*.css"
-stylelint "openwisp_controller/connection/static/connection/css/*.css"
-
-
-# CSS checks
-stylelint "openwisp_controller/connection/static/connection/css/*.css"
 
 openwisp-qa-check \
+  --csslinter
```

Comment:
```suggestion
  --csslinter \
```

Reply:
I would fix and update the changes, thanks.

Reply:
@devkapilbansal, how should i avoid Jshint failing checks?
![Screenshot from 2021-12-09 17-28-47](https://user-images.githubusercontent.com/77020164/145393007-341ed4be-260a-42b5-8060-2bfd6fe0a2be.png)

Reply:
Instead of avoiding, you will have to fix these issues. See which file is causing Jshint to fail, `advanced-mode.js` in this case, make necessary changes and then try again